test/test_common/network_utility.cc

```diff
@@ -0,0 +1,33 @@
+#include "network_utility.h"
+
+#include "common/runtime/runtime_impl.h"
+
+#include <mutex>
```

Comment:
Removed.

I'm so used to the Include-What-You-Use philosophy, which goes along with the bazel approach of separate compilation units, that it no longer occurs to me that there are precompiled headers. In fact, I keep wondering "how is this code compiling?" when I see a symbol used without any corresponding includes. :-)